src/Query.re

```diff
@@ -37,6 +37,7 @@ module Make = (Config: Config) => {
       "data": Js.Nullable.t(Js.Json.t),
       "loading": bool,
       "error": Js.Nullable.t(error),
+      "refetch": _ => 'a,
```

Comment:
actually it can receive the variables as argument

Reply:
it's a nullable of Js.Json.t

Reply:
and it also returns the data as a Js.Json.t, we need still to parse it with Config.parse, like we do here
```
jsResult##data->Js.Nullable.toOption->Belt.Option.map(Config.parse)
```